action.yml

```diff
@@ -9,6 +9,14 @@ inputs:
     default: debian/artifacts
     description: Directory path relative to $GITHUB_WORKSPACE to place the built packages in
     required: true
+  use-debuild:
+    default: 'false'
+    required: true
+    description: If true, debuild is used to create the debian package. Otherwise, dpkg-buildpackage is used
+  debuild-opts:
```

Comment:
It would also be possible to just reuse buildpackage-opts for debuild’s arguments. Their is no real difference in calling debuild or dpkg-buildpackage later in scripts/build_packages.sh.